README.md

```diff
@@ -164,6 +164,7 @@ const config = {
     linkedin: '',
     twitter: '',
     facebook: '',
+    instagram: '',
```

Comment:
Please add instagram to the section `Social Links` also. https://github.com/arifszn/gitprofile/blob/0c2b93ff85da14fd396845450179de603e88e890/README.md?plain=1#L363

Reply:
Here too please
https://github.com/arifszn/gitprofile/blob/0c2b93ff85da14fd396845450179de603e88e890/README.md?plain=1#L375